File: basic/prime.js
import { toInt, print } from '../parv.js';
function isPrime(num) {
  let count = 0;

  if (num < 2) return false;
  else if (num === 2) true;
  for (let i = 2; i <= toInt(num / 2); i++) {
    if (num % i === 0) {
      count++;
    }
  }
  if (count === 0) {
    return true;
  } else return false;
}

let prime = [];

for (let i = 1; i < 100; i++) {
  if (isPrime(i)) prime.push(i);
}
print(prime);

File: cpp_codes/maths/prime.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    int n = 9;
    int count = 0;

    for(int i=2;i<=n/2;i++){
        if(n%i==0){
            count++;
            break;
        }
    }
    if(count==0)
    cout<<n<<" is Prime"<<endl;
    else
    cout<<n<<" is not Prime"<<endl; 
    cout<<count;
}